apps/web/src/components/Dropdown/components/Option.tsx

```diff
@@ -0,0 +1,28 @@
+import { ListboxOption } from "@headlessui/react";
+
+import type { Option as OptionProps } from "../Dropdown";
+
+export const Option: React.FC<OptionProps> = function ({ label, value }) {
+  return (
+    <ListboxOption
+      className={({ active }) =>
+        `relative cursor-pointer select-none py-2 pl-4 pr-4 font-normal ${
+          active
+            ? "bg-controlActive-light dark:bg-controlActive-dark dark:text-content-dark"
+            : "text-contentSecondary-light dark:text-contentSecondary-dark"
+        }`
+      }
+      value={value}
+    >
+      {({ selected }) => (
+        <div className="flex items-center">
+          <span
+            className={`block truncate text-sm ${selected ? "font-bold" : ""}`}
+          >
+            {label}
+          </span>
+        </div>
+      )}
+    </ListboxOption>
+  );
+};
```

Comment:
`active` is deprecated.ts(6385)
listbox.d.ts(62, 9): The declaration was marked as deprecated here

```typescript
type OptionRenderPropArg = {
    /** @deprecated use `focus` instead */
    active: boolean;
    focus: boolean;
    selected: boolean;
    disabled: boolean;
    selectedOption: boolean;
};
```
Here, we replace `active` with `focus` but also modify the content of the `ListboxOption` to accept `children` instead of a label. This still works if you pass a `string` but will work with custom components, too. For example, a component that has a label and an icon. This way, we do not have to modify the dropdown itself to add a dropdown item with an icon.
```suggestion
export const Option: React.FC<OptionProps> = function ({ children, value }) {
  return (
    <ListboxOption
      className={({ focus, selected }) =>
        `flex cursor-pointer select-none items-center px-4 py-2 text-sm
          ${selected ? "font-bold" : "font-normal"}
          ${
            focus
              ? "bg-controlActive-light dark:bg-controlActive-dark dark:text-content-dark"
              : "text-contentSecondary-light dark:text-contentSecondary-dark"
          }
        `
      }
      value={value}
    >
      {children}
    </ListboxOption>
  );
};
```

Reply:
As we agreed, we are going to keep this approach to keep consistency between different variations of the `Option`: with an icon as a prefix, with another component as a suffix... following a slotted approach further if we want to add new elements and keep all `Dropdown` scenarios homogeneous

Regarding the `active` deprecated prop, it has been replaced by `focus` :+1: